sdl_android/src/main/java/com/smartdevicelink/api/ManagerListener.java

```diff
@@ -0,0 +1,15 @@
+package com.smartdevicelink.api;
+
+public interface ManagerListener {
```

Comment:
This is a bit ambiguous in terms of naming convention. Is the intention to use this in other managers as well? If not, it should either be renamed to `SdlManagerListener` / `ISdlManager`, or be moved into the `SdlManager` class and just be titled `Listener`/`Callback` etc.